### ISSUE TITLE
Setup text no longer current

### ISSUE THREAD

**Issue by martijnvv** [r-lib/gmailr#70]
The readme contains a setup instruction that is connected to the former interface of the API setup by Google. It is no longer accurate...

**Comment by mikejacktzen** [comment]
It is one year later. and I can confirm that the API setup instructions no longer match Googles OAuth 2.0

https://developers.google.com/gmail/api/auth/about-auth

Googles app authorization signup interface has changed.  
But even with the change, I **THINK** I successfully obtained my secret foo.json file.

I'm assuming I would **NOT** need the extra human GUI popup google signin step, if I execute the three lines below.

However, when I proceed in R with

```
library(gmailr)

use_secret_file('foo.json')

test_email <- gmailr::mime(
  To = 'foo@foo.com',
  From = "foo@foo.com",
  Subject = 'foo',
  body = 'foo')

gmailr::send_message(test_email)

```
I get an R error complaining about the Google API login

```
Error in gmailr_POST(c("messages", "send"), user_id, class = "gmail_message",  : 
  Gmail API error: 401
  Login Required
In addition: Warning message:
In stop(cond, call. = FALSE) : additional arguments ignored in stop()
```

I'm guessing, either the 'use_secret_file()` function or the action json file due to OAuth 2.0 no longer allows gmailr users to have the desired non sign in functionality.
